composer.json

```
@@ -86,7 +86,7 @@
         "friendsofphp/php-cs-fixer": "~2.18.1",
         "lusitanian/oauth": "~0.8.10",
         "magento/magento-coding-standard": "*",
-        "magento/magento2-functional-testing-framework": "^3.0",
+        "magento/magento2-functional-testing-framework": "dev-package/monolog",
```

```suggestion
        "magento/magento2-functional-testing-framework": "dev-develop",
```
And please remove the corresponding declaration from the repositories section

---

Hello @fascinosum 
There are other changes and I changed the repository and branch for it.